bin/coreos-install

```diff
@@ -334,6 +343,31 @@ if [[ -n "${CLOUDINIT}" ]] || [[ -n "${COPY_NET}" ]]; then
     umount "${WORKDIR}/rootfs"
 fi
 
+if [[ -n "${IGNITION}" ]]; then
+  # The OEM partition should be #3 but make no assumptions here!
+  # Also don't mount by label directly in case other devices conflict.
+  OEM_DEV=$(blkid -t "LABEL=OEM" -o device "${DEVICE}"*)
+
+  if [[ -z "${OEM_DEV}" ]]; then
+    echo "Unable to find new OEM partition on ${DEVICE}" >&2
+    exit 1
+  fi
+
+  mkdir -p "${WORKDIR}/oemfs"
+
+  case $(blkid -t "LABEL=OEM" -o value -s TYPE "${OEM_DEV}") in
+    "btrfs") mount -t btrfs -o subvol=root "${OEM_DEV}" "${WORKDIR}/oemfs" ;;
+    *)       mount "${OEM_DEV}" "${WORKDIR}/oemfs" ;;
+  esac
+  trap "umount '${WORKDIR}/oemfs' && rm -rf '${WORKDIR}'" EXIT
+
+  echo "Installing Ignition config ${IGNITION}..."
+  cp "${IGNITION}" "${WORKDIR}/oemfs/${IGNITION}"
+  echo  "set linux_append=coreos.config.url=oem:///${IGNITION}" > "${WORKDIR}/oemfs/grub.cfg"
```

Comment:
The value for linux_append will need to be quoted. It might also need to be a bit smarter eventually to avoid clobbering existing settings, but it's probably okay for now.